post_tags/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .request import search_post_by_tag
```

Comment:
we really need to group these at some point....

Reply:
good idea